README.md

```diff
@@ -61,15 +61,15 @@ Follow these steps to create a container that you can than deploy as a Service o
 ```sh 
 minikube start --cpus 4 --memory 4096 --addons registry --insecure-registry "10.0.0.0/24"
 ```
-2. Create a namespace for the building phase
+2. Create a namespace for your workflows
 
 ```sh
-kubectl create namespace kogito-builder
+kubectl create namespace kogito-workflows
 ```
 
 3. Create a secret
```

Comment:
Please add the purpose of this secret, e.g. pull built images to this registry.etc..

Reply:
Added an additional description